VocaDb.Migrations/Migrations.cs

```diff
@@ -1,1805 +1,1866 @@
-using System.Data;

```

Comment:
I wonder, why is the whole file changed? Line endings?

Reply:
I've resolved conflicts in Migrations.cs on GitHub and I'm not sure why the whole file has changed...

Reply:
Fixed.